Mathlib/SetTheory/Ordinal/FixedPointApproximants.lean

```diff
@@ -110,7 +110,7 @@ theorem lfpApprox_add_one (h : x ≤ f x) (a : Ordinal) :
     simp only [Set.mem_setOf_eq]
     use a
 
-/-- The ordinal approximants of the least fixed point are stabilizing
+/-- The ordinal-indexed approximantions of the least fixed point are stabilizing
```

Comment:
```suggestion
/-- The approximations of the least fixed point stabilize at a fixed point of `f`
```
I don't think it's necessary to reiterate that we're talking about ordinals throughout the entire file.